File: class.java
class Animal {
    public String name;

    public String getFeatures() {...}
    public String run() {
        return "I can run."
    }
    public String run(String name) {
        return "I am " + name + ", I can run."
    }
    public String jump() {
        return "I can jump."
    }
}

class people extends Animal{
    public String talk(name) {
        return "Hello, I am " + name + "."
    }
    public String run(String name) {
        return "I am " + name + ", I can run."
    }
}

public static void main() {
    Animal dog = new Animal();
    Animal cat = new Animal();

    dog.run("white");
    cat.jump();
}

File: class.py
class Animal:
    def __init__(self, name):
        self.name = name

    def getFeatures():
        pass

    def run(self):
        return "I can run."
    
    def jump(self):
        return "I can jump."


class Dog(Animal):
    def __init__(self, name):
        self.name = name

    def sit(self):
        return "I am Dog, I can sit."
    
    def run(self):
        return "I am " + self.name + ", I can runing."


a = Animal("a")
b = Animal('b')

a.run()
b.jump()


c = Dog('lucky')
c.run()
